### PR TITLE
fix: add Amandeep404

### LINKED ISSUE #1278: Amandeep Singh
Amandeep404 (Amandeep Singh)

### PR DESCRIPTION
**Pull request by Amandeep404**
resolves #1278

## Checklist for profile addition

- [x] Name the PR `fix: add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

---

## Other additions/changes

- [x] Changes made in this PR are mentioned below
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.
- [x] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

**Comment by Amandeep404** [comment]
why my code is failing during the prettier check? how can I solve this issue? any guidance?

**Comment by RayanAbid** [comment]
I think you have to remove the ``Other additions/changes `` Section

File: profiles/Amandeep.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/107784525/182681880-2fa3d360-76b2-4a96-8bb1-20978f31ef8f.png",
+  "issueId": 1278,
+  "name": "Amandeep Singh",
+  "username": "Amandeep404"
+}
```
Comment (Panquesito7) [inline profiles/Amandeep.json:6]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
This should also fix the Prettier (or Continuous Integration) issues.
```suggestion
}

```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Comment by Amandeep404** [comment]
thanks :)

**Review by Panquesito7 — APPROVED** [review]
Thanks for contributing! 🎉


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Maintainable

**Pull request merged**